src/main/resources/application-dev.yaml

```diff
@@ -23,31 +16,9 @@ spring:
     change-log: db/dbchangelog-dev.yaml
     drop-first: true
 
-  #Jpa/Hibernate:
-  jpa:
-    open-in-view: false
-    generate-ddl: false
-    hibernate:
-      ddl-auto: validate
-    database-platform: org.hibernate.dialect.MySQL8Dialect
-    #do debugowania sql
-    show-sql: true
-    properties:
-      hibernate:
-        format_sql: true
-
```

Comment:
34-37 Lines was was only in dev profile.  Do we need them in general profiles?

Reply:
You're right, I'll correct it.